src/main/memorygame/app.cljs

```diff
@@ -6,10 +6,11 @@
 (def a (r/atom nil))
 (def billedrækken-2 (r/atom (zipmap (range 1 17) (shuffle (concat (range 1 9) (range 1 9))))))
 (def billedrækken (r/atom (zipmap (range 1 17) (concat (range 1 9) (range 1 9)))))
+(def allerede-vundet (r/atom ()))
 (def tilstand (atom {
                      :kort         nil
 
-                     :næstespiller :player1
+                     :næstespiller 0
```

Comment:
Jeg tror det vil være bedre med at beholde :player1. Så kan man lave et opslag ved den værdi

Reply:
The game logic I wrote uses even or odd integers to track the player and was a cunning way to keep the logic simple. I propose we use (if (even? (@tilstand :næstespiller)) "Player 1" "Player 2") in the opslag.

Reply:
But, If you guess correct you'll keep your turn. So i don't think  you can't just use that

Reply:
I have already worked that into the solution, the whole thing functions. Ie, you keep your turn etc.

Reply:
logic here: 
(defn spil [event]
      (let [[placering kort] event]
           (if (nil? (@tilstand :kort))
             (do
               (swap! tilstand assoc-in [:kort] kort))
             (do
               (if (and (= kort (@tilstand :kort)) (not (some #(= kort %) @allerede-vundet)))
                 (do
                   (swap! tilstand update-in [(if
                                              (even? (@tilstand :næstespiller)) :player1 :player2) :points] inc)
                   (swap! allerede-vundet conj kort)
                   (print @allerede-vundet))
                 (swap! tilstand update-in [:næstespiller] inc))
               (swap! tilstand assoc-in [:kort] nil))))
      (print @tilstand))